src/olympia/reviewers/models.py

```diff
@@ -1148,7 +1148,6 @@ def get_queryset(self):
 class RereviewQueueTheme(ModelBase):
     theme = models.ForeignKey(Persona)
     header = models.CharField(max_length=72, blank=True, default='')
-    footer = models.CharField(max_length=72, blank=True, default='')
```

Comment:
This needs a migration in `src/olympia/migrations/` that removes this field from the database. This should not be done in this PR though but as a follow-up. Can you file an issue about that?